cmd/convert.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kong/deck/convert"
+	"github.com/spf13/cobra"
+)
+
+var (
+	convertCmdSourceFormat      string
+	convertCmdDestinationFormat string
+	convertCmdInputFile         string
+	convertCmdOutputFile        string
+)
+
+// convertCmd represents the convert command
+var convertCmd = &cobra.Command{
+	Use:   "convert",
+	Short: "Convert files in one format to another format",
+	Long: `Convert command converts files representing configuration in one format
+to another compatible format.
+and usage of using your command. For example: a configuration for 'kong-gateway'
+can be converted into 'konnect' configuration file.`,
+	Args: validateNoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		sourceFormat, err := convert.ParseFormat(convertCmdSourceFormat)
+		if err != nil {
+			return err
+		}
+		destinationFormat, err := convert.ParseFormat(convertCmdDestinationFormat)
+		if err != nil {
+			return err
+		}
```

Comment:
Makes total sense.
I'm a bit torn: wouldn't that mean `convert` package is now aware of how `cmd` is using it? Maybe that's okay? How do you go about reasoning against/for this?

Reply:
You're right - the orthodox way would IMO be to implement a wrapper type for Format that implements the interface. 

But Go's duck-typed interfaces allow us to take a shortcut here, and I think that shortcut is fine (because, worst case, somebody will need to implement that wrapper if they change the contract of `Format`)

In Java that wouldn't fly because it would need `Format` to explicitly implement that interface.

Reply:
This turned out to be significantly more complicated (layers of indirection, unnecessary FlagSet types) and I saw no real benefit of it so I gave up. Let me know if this is a blocker.

Reply:
Let's pass on this.